src/EnergyPlus/MixedAir.cc

```diff
@@ -3674,6 +3674,24 @@ namespace MixedAir {
         }
         this->MechVentOAMassFlowRequest = MechVentOAMassFlow;
         //****** use greater of Mechanical Ventilation Outside Air fraction and OutAirMinFrac
+        if (OutAirMinFrac > MechVentOutsideAirMinFrac) {
+            if (!WarmupFlag) {
+                if (CountMechVentFrac == 0) {
+                    ++CountMechVentFrac;
+                    ShowWarningError("Min OA fraction > Mechanical vetilation min OA fraction for OA controller=" + this->Name + ", Min OA fraction is used");
+                    ShowWarningError("Therefore, Mechanical vetilation is not effective as desired.");
+                    ShowContinueErrorTimeStamp("Min OA fraction = " + RoundSigDigits(OutAirMinFrac, 4) +
+                                               ", Min Mech Vent OA fraction = " + RoundSigDigits(MechVentOutsideAirMinFrac, 4));
```

Comment:
How about this error text:
```
                    ShowWarningError(RoutineName+"Minimum OA fraction > Mechanical Ventilation Controller request for Controller:OutdoorAir=" + this->Name + ", Min OA fraction is used.");
                    ShowContinueError("This may be overriding desired ventilation controls. Check inputs for Minimum Outdoor Air Flow Rate, Minimum Outdoor Air Schedule Name and Controller:MechanicalVentilation");
                    ShowContinueErrorTimeStamp("Minimum OA fraction = " + RoundSigDigits(OutAirMinFrac, 4) +", Mech Vent OA fraction = " + RoundSigDigits(MechVentOutsideAirMinFrac, 4));
```

Reply:
@mjwitte Your suggestion is much better than mine. Use yours. Thanks,